x-pack/plugins/index_lifecycle_management/__jest__/client_integration/app/app.helpers.ts

```diff
@@ -5,31 +5,11 @@
  * 2.0.
  */
 
-import React from 'react';
 import { act } from 'react-dom/test-utils';
+import { HttpSetup } from '@kbn/core/public';
 import { registerTestBed, TestBed, TestBedConfig } from '@kbn/test-jest-helpers';
-import { docLinksServiceMock, executionContextServiceMock } from '@kbn/core/public/mocks';
-import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
-import { licensingMock } from '@kbn/licensing-plugin/public/mocks';
-import { createBreadcrumbsMock } from '../../../public/application/services/breadcrumbs.mock';
 import { App } from '../../../public/application/app';
-
-const breadcrumbService = createBreadcrumbsMock();
-
-const AppWithContext = (props: any) => {
```

Comment:
This has been refactored into a `WithAppDependencies` HoC that is now defined in `setup_environment.tsx` so that it can be reused by other test files